charts/opentelemetry-ebpf-instrumentation/values.yaml

```diff
@@ -293,9 +293,9 @@ k8sCache:
     # -- K8s Cache image registry (defaults to ghcr.io)
     registry: "ghcr.io"
     # -- K8s Cache image repository.
-    repository: otel/opentelemetry-ebpf-k8s-cache
+    repository: open-telemetry/opentelemetry-ebpf-instrumentation/opentelemetry-ebpf-k8s-cache
     # -- (string) K8s Cache image tag. When empty, the Chart's appVersion is used.
-    tag: main
+    tag: v0.1.0
```

Comment:
do we not want to be using the Chary.yaml app version here per the comment?

Reply:
Are you sure the registry address `open-telemetry/opentelemetry-ebpf-instrumentation/` here is correct? I think the original `otel/` is correct.

following https://hub.docker.com/r/otel/opentelemetry-ebpf-k8s-cache

Reply:
@JaredTan95  yeah, its a different image repo and name when downloading from ghcr.io instead of docker.io

Reply:
> i see its done like that in the collector chart, i can also do something like this, maybe in a different PR tho? since this fixes the chart to pull the correct images

I created another issue to track it. see https://github.com/open-telemetry/opentelemetry-helm-charts/issues/1931